### PR TITLE
Close changelog for 7.6.0

### PR DESCRIPTION
**Pull request by mikemadden42**

### CODE REVIEW

**Comment by mikemadden42** [comment]
jenkins retest this please

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -3,6 +3,112 @@
...
 
+[[release-notes-7.6.0]]
+=== Beats version 7.6.0
+https://github.com/elastic/beats/compare/v7.5.1...v7.6.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Allow Metricbeat's beat module to read monitoring information over a named pipe or unix domain socket. {pull}14558[14558]
```
Comment (urso) [inline CHANGELOG.asciidoc]: I don't think this is a breaking change.

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -3,6 +3,112 @@
...
+- Remove version information from default ILM policy for improved upgrade experience on custom policies. {pull}14745[14745]
+- Running `setup` cmd respects `setup.ilm.overwrite` setting for improved support of custom policies. {pull}14741[14741]
+- Libbeat: Do not overwrite agent.*, ecs.version, and host.name. {pull}14407[14407]
```
Comment (urso) [inline CHANGELOG.asciidoc]: This is rather a fix. like "Fix existing agent.*, ecs.version, and host.name fields getting overwritten by Beats if they are already present in the original event".

File: CHANGELOG.asciidoc [hunk 3]
```diff
@@ -3,6 +3,112 @@
...
+- Libbeat: Cleanup the x-pack licenser code to use the new license endpoint and the new format. {pull}15091[15091]
```
Comment (urso) [inline CHANGELOG.asciidoc]: wording? I guess this PR replaces the url `/_xpack/license` with `/_license`.

File: CHANGELOG.asciidoc [hunk 4]
```diff
@@ -3,6 +3,112 @@
...
+- Users can now specify `monitoring.cloud.*` to override `monitoring.elasticsearch.*` settings. {issue}14399[14399] {pull}15254[15254]
```
Comment (urso) [inline CHANGELOG.asciidoc]: This is no breaking change.

File: CHANGELOG.asciidoc [hunk 5]
```diff
@@ -3,6 +3,112 @@
...
+- Refactor metadata generator to support adding metadata across resources {pull}14875[14875]
+- Update to ECS 1.4.0. {pull}14844[14844]
```
Comment (urso) [inline CHANGELOG.asciidoc]: @adriansr @webmat Is this really a breaking change (like ECS 1.4 not being BC to 1.2?)

File: CHANGELOG.asciidoc [hunk 6]
```diff
@@ -3,6 +3,112 @@
...
+
+- CEF extensions are now mapped to the data types defined in the CEF guide. {pull}14342[14342]
+
+*Journalbeat*
+
+- Remove broken dashboard. {pull}15288[15288]
+
+*Metricbeat*
+
+- Update cloudwatch metricset mapping for both metrics and dimensions. {pull}15245[15245]
```
Comment (urso) [inline CHANGELOG.asciidoc]: @kaiyan-sheng Is this really a breaking change?

File: CHANGELOG.asciidoc [hunk 7]
```diff
@@ -3,6 +3,112 @@
...
+- GA the `script` processor. {pull}14325[14325]
+- Add `fingerprint` processor. {issue}11173[11173] {pull}14205[14205]
+- Add support for API keys in Elasticsearch outputs. {pull}14324[14324]
+- Add consumer_lag in Kafka consumergroup metricset {pull}14822[14822]
+- Make use of consumer_lag in Kafka dashboard {pull}14863[14863]
+- Refactor kubernetes autodiscover to enable different resource based discovery {pull}14738[14738]
+- Add `add_id` processor. {pull}14524[14524]
+- Enable TLS 1.3 in all beats. {pull}12973[12973]
+- Spooling to disk creates a lockfile on each platform. {pull}15338[15338]
+- Fingerprint processor adds a new xxhash hashing algorithm {pull}15418[15418]
```
Comment (urso) [inline CHANGELOG.asciidoc]: the processor and the enhancement are added in the same release. I think we can remove this line.

File: CHANGELOG.asciidoc (hunk 5 above)
Comment (adriansr) [reply CHANGELOG.asciidoc]: It introduced some breaking changes in Packetbeat, but I ended up adding dedicate entries below. Feel free to move it around.

File: CHANGELOG.asciidoc (hunk 3 above)
Comment (ph) [reply CHANGELOG.asciidoc]: Agree with the wording changes here ^

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (ph) [reply CHANGELOG.asciidoc]: No its not a breaking change its an enhacement.,

File: CHANGELOG.asciidoc (hunk 6 above)
Comment (kaiyan-sheng) [reply CHANGELOG.asciidoc]: @urso Yes, I believe so. cloudwatch metricset is in GA and we changed the mapping including metric names and dimension names.

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (mikemadden42) [reply CHANGELOG.asciidoc]: Thanks.  I'll move it to the _Added_ section.

File: CHANGELOG.asciidoc (hunk 2 above)
Comment (mikemadden42) [reply CHANGELOG.asciidoc]: I changed the wording as suggested.

File: CHANGELOG.asciidoc (hunk 3 above)
Comment (mikemadden42) [reply CHANGELOG.asciidoc]: Thanks. I changed the wording as suggested.

File: CHANGELOG.asciidoc (hunk 4 above)
Comment (mikemadden42) [reply CHANGELOG.asciidoc]: I moved it to the _Added_ section.

File: CHANGELOG.asciidoc [hunk 8]
```diff
@@ -3,6 +3,112 @@
...
+
+*Affecting all Beats*
+
+- Allow Metricbeat's beat module to read monitoring information over a named pipe or unix domain socket. {pull}14558[14558]
+- Remove version information from default ILM policy for improved upgrade experience on custom policies. {pull}14745[14745]
+- Running `setup` cmd respects `setup.ilm.overwrite` setting for improved support of custom policies. {pull}14741[14741]
+- Libbeat: Do not overwrite agent.*, ecs.version, and host.name. {pull}14407[14407]
+- Libbeat: Cleanup the x-pack licenser code to use the new license endpoint and the new format. {pull}15091[15091]
+- Users can now specify `monitoring.cloud.*` to override `monitoring.elasticsearch.*` settings. {issue}14399[14399] {pull}15254[15254]
+- Refactor metadata generator to support adding metadata across resources {pull}14875[14875]
```
[the remark this answers is not in the document]
Comment (mikemadden42) [reply CHANGELOG.asciidoc]: I moved this to the _Added_ section.

File: CHANGELOG.asciidoc (hunk 5 above)
Comment (mikemadden42) [reply CHANGELOG.asciidoc]: I moved it to the _Added_ section.

File: CHANGELOG.asciidoc (hunk 7 above)
Comment (mikemadden42) [reply CHANGELOG.asciidoc]: I removed the line.

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (ycombinator) [reply CHANGELOG.asciidoc]: @mikemadden42 Looks like this still needs to be moved?

File: CHANGELOG.next.asciidoc [hunk 9]
```diff
@@ -112,24 +71,11 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
...
-- Add a friendly log message when a request to docker has exceeded the deadline. {pull}15336[15336]
 - Decouple Debug logging from fail_on_error logic for rename, copy, truncate processors {pull}12451[12451]
 - Allow a beat to ship monitoring data directly to an Elasticsearch monitoring cluster. {pull}9260[9260]
 - Updated go-seccomp-bpf library to v1.1.0 which updates syscall lists for Linux v5.0. {pull}11394[11394]
 - add_host_metadata is no GA. {pull}13148[13148]
 - Add `providers` setting to `add_cloud_metadata` processor. {pull}13812[13812]
-- GA the `script` processor. {pull}14325[14325]
-- Add `fingerprint` processor. {issue}11173[11173] {pull}14205[14205]
-- Add support for API keys in Elasticsearch outputs. {pull}14324[14324]
-- Ensure that init containers are no longer tailed after they stop {pull}14394[14394]
```
Comment (ycombinator) [inline CHANGELOG.next.asciidoc]: @mikemadden42 I can't find this line in the `CHANGELOG.asciidoc` file. Shouldn't it be there?

Comment (ycombinator) [reply CHANGELOG.next.asciidoc]: Never mind, looks like this was released in 7.5.0.

File: CHANGELOG.asciidoc (hunk 8 above)
[the remark this answers is not in the document]
Comment (exekias) [reply CHANGELOG.asciidoc]: sorry I lost this notification, you are right, this was good to  move

**Pull request merged**